### DISCUSSION TITLE
New here

### DISCUSSION THREAD

**Discussion by Alien293** [community/community#43772] [New to GitHub]
Hi to all...
Totally new here, new and lost....
I signed up for SQL Server Management Studio online classes and it seems so complicated. Or is it to easy for GitHub?
Anyone here with good tips maybe?
Thanks in advance.

**Comment by queenofcorgis** [comment]
Hi @Alien293,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on SQL Server Management Studio. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!

**Reply by Alien293** [reply]
Thank you :)

**Comment by melvincwng** [comment]
Hey @Alien293 :wave:, welcome to GitHub Community!

For starters, I would suggest you familiarize with the GitHub platform using [GitHub Skills](https://skills.github.com/). There's basically a bunch of mini-courses which you can do to get yourself up and running with GitHub.

After which, you would have to delve deeper in other topics like Git, various frontend languages & tools, various backend languages/tools, databases, cloud-based technologies & various additional tooling. There's many things to learn in software development, and it may seem overwhelming at times, but my advice to you is to **take it slow, and keep on learning**.

Even as you start working in the industry, you'll soon realize there's only so much that one can ever know. We all are continuously learning along the way. So, just **keep on learning :)**, and remember to **always share information with your peers**.

All the best in your software development journey! Hope this reply has been helpful to you :)

Good luck, and take care :smile:

**Reply by Alien293** [reply]
Hey @melvincwng 
Thank you so much for your reply and for your suggestions. 
You guys here are all so nice :)
